yarn.lock

```diff
@@ -1,16608 +0,0 @@
-# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```

Comment:
this needs to be reverted back

Reply:
There seems to be varying practices regarding lock-files for libraries. I thought I might have added it by accident. Will restore :)